### PR TITLE
Commercial components position AB tests

### PR DESCRIPTION
**Pull request by uplne**
We have two server side AB tests to run. Both will run on 3% of audience.
1. Commercial high relevance component is moved from top position to above the most popular container
2. Commercial high relevance component is moved from top position to above the most popular container and outbrain is moved to second position (below related content)

If the related is switched off outbrain is moved to second position below onward.

CC @rich-nguyen @Calanthe

### CODE REVIEW

**Comment by Calanthe** [comment]
It looks fine to me, but maybe let's wait for some backend dev opinion.

File: common/app/views/fragments/contentFooter.scala.html [hunk 1]
```diff
@@ -7,28 +9,55 @@
     @* majority of footer components we don't want to appear on advertisement feature articles *@
     @if(!content.isAdvertisementFeature) {
 
-        @if(!content.shouldHideAdverts) {
+        @if(!content.shouldHideAdverts && (!mvt.CMHRTest.isParticipating && !mvt.CMOutbrainTest.isParticipating)) {
```
Comment (rich-nguyen) [inline common/app/views/fragments/contentFooter.scala.html]: `!(content.shouldHideAdverts || mvt.CMHRTest.isParticipating || mvt.CMOutbrainTest.isParticipating)`

File: common/app/views/fragments/contentFooter.scala.html [hunk 2]
```diff
@@ -7,28 +9,55 @@
...
+                }
+                case _ => { }
+            }
+        }
 
         @if(content.isCommentable) {
             <div class="js-repositioned-comments content__repositioned-comments"></div>
         }
 
+        @if(!content.shouldHideAdverts && (!mvt.CMHRTest.isParticipating || !mvt.CMOutbrainTest.isParticipating)) {
```
Comment (rich-nguyen) [inline common/app/views/fragments/contentFooter.scala.html]: same as above

**Comment by rich-nguyen** [comment]
:+1: great stuff

**Pull request merged**